app/src/main/java/org/ieeeguc/ieeeguc/fragments/user/UserShow.java

```diff
@@ -6,19 +6,45 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.google.gson.Gson;
 
 import org.ieeeguc.ieeeguc.R;
+import org.ieeeguc.ieeeguc.models.User;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
 
 /**
  * A simple {@link Fragment} subclass.
  */
 public class UserShow extends Fragment {
 
+    private User user;
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_user_show, container, false);
+        View view = inflater.inflate(R.layout.fragment_user_show, container, false);
+        user = new Gson().fromJson(getArguments().getString("user_show"), User.class);

```

Comment:
call the API that gets a user and get the info of the logged in user from the server using the id of the logged in user